docs/components/guide-examples/examples/PolarCoordinates.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import { Mafs, Line, Coordinates, useMovablePoint, Plot } from "mafs";
+
+export default function PolarCoordinate() {
+  const viewBox = { x: [-2, 2], y: [-4, 4] };
```

Comment:
This is kind of silly but this is technically the wrong type, since TypeScript is inferring these as arrays of numbers rather than tuples of numbers. The docs site preview should build successfully if you either add `as const` here, or if you inline this variable directly into the Mafs props.